robottelo/ui/template.py

```diff
@@ -17,9 +17,45 @@ def _search_locator(self):
         """Specify locator for Template entity search procedure"""
         return locators['provision.template_select']
 
+    def _configure_template(self, locations=None, organizations=None,
+                            select=None, os_list=None, new_os_list=None,
+                            hostgroup=None, environment=None):
+        """Configures different entities for specific template"""
+        loc = tab_locators
+        if locations:
+            self.configure_entity(
+                locations,
+                FILTER['template_location'],
+                tab_locator=loc['provision.tab_locations'],
+                entity_select=select
+            )
+        if organizations:
+            self.configure_entity(
+                organizations,
+                FILTER['template_org'],
+                tab_locator=loc['provision.tab_organizations'],
+                entity_select=select,
+            )
+        if os_list or new_os_list:
+            self.configure_entity(
+                os_list,
+                FILTER['template_os'],
+                tab_locator=tab_locators['provision.tab_association'],
+                new_entity_list=new_os_list
+            )
```

Comment:
Iterating through a dict keys would be better option, but that's definitely low-priority nitpick